org.eclipse.xtext.ide.tests/model/generated/IndentationAwareUiTestLanguage.ecore

```diff
@@ -9,12 +9,12 @@
         eType="#//OtherTreeNode" containment="true"/>
   </eClassifiers>
   <eClassifiers xsi:type="ecore:EClass" name="TreeNode">
-    <eStructuralFeatures xsi:type="ecore:EAttribute" name="name" eType="ecore:EDataType http://www.eclipse.org/emf/2002/Ecore#//EString"/>
+    <eStructuralFeatures xsi:type="ecore:EAttribute" name="name" eType="ecore:EDataType platform:/resource/org.eclipse.emf.ecore/model/Ecore.ecore#//EString"/>
     <eStructuralFeatures xsi:type="ecore:EReference" name="children" upperBound="-1"
         eType="#//TreeNode" containment="true"/>
   </eClassifiers>
   <eClassifiers xsi:type="ecore:EClass" name="OtherTreeNode">
-    <eStructuralFeatures xsi:type="ecore:EAttribute" name="name" eType="ecore:EDataType http://www.eclipse.org/emf/2002/Ecore#//EString"/>
```

Comment:
am not sure if this change is good. cc @szarnekow

Reply:
@kthoms @szarnekow any ideas?

Reply:
I think it's ok here - the test lang was probably not regenerated as part of the entire test lang generator before.